RafRate.API/Configuration/Extentions/ServicesExtentions.cs

```diff
@@ -1,14 +1,14 @@
-﻿using RafRate.Core.Interfaces;
-using RafRate.Core.Services;
+﻿using RafRate.Core.Services;
+using RafRate.Core.Interfaces;
 
-namespace RafRate.API.Extentions
+namespace RafRate.API.Configuration.Extentions
```

Comment:
you can just write `namespace MyNamespace;` and have 1 less `{ }` and indentation level